discord/application_command.go

```diff
@@ -435,3 +444,131 @@ func (c MessageCommand) CreatedAt() time.Time {
 }
 
 func (MessageCommand) applicationCommand() {}
+
+var _ ApplicationCommand = (*EntryPointCommand)(nil)
+
+type EntryPointCommand struct {
+	id                       snowflake.ID
+	applicationID            snowflake.ID
+	guildID                  *snowflake.ID
+	name                     string
+	nameLocalizations        map[Locale]string
+	nameLocalized            string
+	defaultMemberPermissions Permissions
+	dmPermission             bool
+	nsfw                     bool
+	integrationTypes         []ApplicationIntegrationType
+	contexts                 []InteractionContextType
+	version                  snowflake.ID
+	handler                  *EntryPointCommandHandlerType
```

Comment:
if you create an entry point command without providing the `handler` field, subsequent fetches of the command will not have the field (I have no clue)

Reply:
pretty sure this shouldn't matter then?
since the enum is 1 & 2 0 will indicate that it's missing and we don't need a pointer